src/main/fc/fc_msp_box.c

```diff
@@ -80,7 +80,7 @@ static const box_t boxes[CHECKBOX_ITEM_COUNT] = {
     { BOXPREARM, "PREARM", 36 },
     { BOXBEEPGPSCOUNT, "BEEP GPS SATELLITE COUNT", 37 },
     { BOX3DONASWITCH, "3D ON A SWITCH", 38 },
-
+    { BOXMOTORBEEPER, "MOTOR BEEPER", 39 },
```

Comment:
I would probably call this 'motor beacon', since that's what was decided that it should be.

Reply:
Yes.  Will do